### DISCUSSION TITLE
whats the meaning of det_count in fan_beam projection ?

### DISCUSSION THREAD

**Discussion by argman** [astra-toolbox/astra-toolbox#515] [Q&A] [answered]
i want to know how the projection is computed in astra, in matlab there is a clear document like https://ww2.mathworks.cn/help/images/fan-beam-projection-data.html

**Comment by askorikov** [comment, marked as answer]
Hi! `det_count` is the number of pixels in the (1D) detector. I'm not sure what you mean by "how the projection is computed", but you can check the Astra Toolbox paper https://doi.org/10.1364/OE.24.025129
